yt/config.py

```diff
@@ -1,7 +1,9 @@
 import os
 import warnings
 
-import toml
+# TODO: import tomllib from the standard library instead in Python >= 3.11
```

Comment:
This is likely going to break some developer installations where folks do an in-place without dependency checks, so we should send a note to the mailing list.

Reply:
You got it.